### PR TITLE
MAT-4633 measure group page new design components

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-4633](https://jira.cms.gov/browse/MAT-4633)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [x] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by sb-cecilialiu** [comment]
image

**Review by chubert-sb — DISMISSED** [review]

**Comment by RohitKandimalla** [comment]
> image

Not exactly sure what is the expectation from the story. but 
AC 1.5 specifies  - Update color and font of the action buttons (Save, Discard Changes, Cancel) - add specific link.

And looks like this AC is not yet met.

File: src/components/measureGroups/MeasureGroups.tsx [hunk 1]
```diff
@@ -44,12 +42,11 @@ import {
...
 
 const Grid = styled.div(() => [tw`grid grid-cols-4 ml-1 gap-y-4`]);
-const Content = styled.div(() => [tw`col-span-3`]);
+const Content = styled.div(() => [tw`col-span-3 pl-4 pr-4`]);
 const Header = styled.section`
-  background-color: #f2f5f7;
-  padding: 1em 3em;
   border-bottom: solid 1px rgba(80, 93, 104, 0.2);
 `;
 const Title = styled.h1`
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx]: Looks like we need a h3 and default size and also the color should be #242424. Also the header section supposed to have help text 
* Indicates required field

Comment (sb-cecilialiu) [reply src/components/measureGroups/MeasureGroups.tsx]: @RohitKandimalla Thanks Rohit. I just pushed in the change for this.

**Comment by sb-cecilialiu** [comment]
new screen shot: 
image

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**